testing/gocryptfs/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor: André Klitzing <aklitzing@gmail.com>
+# Maintainer: André Klitzing <aklitzing@gmail.com>
+pkgname=gocryptfs
+pkgver=1.3
+pkgrel=0
+pkgdesc="Encrypted overlay filesystem written in Go"
+url="https://nuetzlich.net/gocryptfs/"
+arch="all"
+license="MIT"
+makedepends="go libressl-dev fuse-dev bash"
+source="$pkgname-$pkgver.tar.gz::https://github.com/rfjakob/$pkgname/archive/v$pkgver.tar.gz"
+builddir="$srcdir/"
+
+build() {
+	cd "$builddir"
+
+	export GOPATH="$PWD/GO"
+	mkdir -p $GOPATH/src/github.com/rfjakob
+	ln -sf $PWD/$pkgname-$pkgver $GOPATH/src/github.com/rfjakob/$pkgname
+
+	go get -d github.com/rfjakob/gocryptfs
```

Comment:
I'm not a big fan of this as it bypasses abuild's source and checksum mechanism. Can't `go-fuse` be included in source, or even better, be made a separate package?